parlai/core/dict.py

```diff
@@ -99,6 +100,10 @@ def add_cmdline_args(argparser):
             '--dict-minfreq', default=DictionaryAgent.default_minfreq,
             type=int,
             help='minimum frequency of words to include them in sorted dict')
+        dictionary.add_argument(
+            '--dict-maxsortedsize', default=DictionaryAgent.default_maxsortedsize,
```

Comment:
maybe just `--dict-maxtokens`? @jaseweston

Reply:
yes maxtokens is fine although im not sure how we do our spacing

btw, the switch between "_"  and "-" which i guess was alex's decision? gets me like several times a day..